content/swift/concepts/optionals/optionals.md

```diff
@@ -0,0 +1,94 @@
+---
+Title: 'Optionals'
+Description: 'Optionals are types that deal with scenarios involving the potential presence or absence of a value (nil).'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Optionals'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+Optionals are types that deal with scenarios involving the potential presence or absence of a value (nil).
+
+## Syntax
+
+In Swift, a non-optional variable can't be set to nil. Rather, the optional type must be defined using the added question mark `?`.
+
+Below, a variable `myInt` is declared and set to the optional `Int` type, `Int?`. It can either be equal to any integer or nil.
+
+```swift
+var maybeInt: Int? = nil
+```
+
+**Note:** If `myInt` is not assigned the value of nil, it will default to nil.
+
+## Unwrapping Optionals
+
+Optional types can be unwrapped if it's certain that a value is present. Below are some ways this can be done.
```

Comment:
Would it also be correct to assume that values of an optional type must be unwrapped before they can be used?

Reply:
I would say so. I also added a line in regard to this